index.html

```diff
@@ -34,6 +34,19 @@
     </div>
   </div>
 
+  <script src="https://www.gstatic.com/firebasejs/4.8.1/firebase.js"></script>
+  <script>
+    // Initialize Firebase
+    var config = {
+      apiKey: "AIzaSyC3S4hjFO1HLOrhhUDkwqkbsnxPWaQyjjg",
```

Comment:
Defo put these in a .env file, as this repo is public someone could find these and hammer your firebase API.

Reply:
Good shout. Will have to look into that...